### PR TITLE
Add home link to schedulator logo

### PR DESCRIPTION
**Pull request by nicudy**
adds link to home when the schedulator logo is clicked

### CODE REVIEW

File: src/components/Nav.tsx [hunk 1]
```diff
@@ -18,7 +18,11 @@ import {
 import React, { useEffect } from "react";
 import { gql, useMutation } from "@apollo/client";
 import { HamburgerIcon } from "@chakra-ui/icons";
-import { Link as ReactRouterLink, useNavigate } from "react-router-dom";
+import {
+  Link as ReactRouterLink,
+  Link as Link2,
```
Comment (jayden-chan) [inline src/components/Nav.tsx]: What's going on with this lol, you should be able to just replace all usage of `Link2` with `ReactRouterLink` and then remove the `Link2` alias

File: src/components/Nav.tsx [hunk 2]
```diff
@@ -149,14 +153,22 @@ export const NavHeader = () => {
...
-            alt="schedulator logo"
-            minWidth="40px"
-            padding="5px"
-          ></Image>
-        </Box>
+        <Link2
+          to={
+            user?.roles.includes("admin") || user?.roles.includes("user")
+              ? "/login"
+              : "/"
```
Comment (jayden-chan) [inline src/components/Nav.tsx]: Is it just me or is this condition backwards? We want it to go to `/login` when there is no user, not when there is a user.

File: src/components/Nav.tsx (hunk 1 above)
Comment (nicudy) [reply src/components/Nav.tsx]: Haha I don't know what I was thinking. Will change this

File: src/components/Nav.tsx (hunk 2 above)
Comment (nicudy) [reply src/components/Nav.tsx]: when logged in, navigation to /login will redirect you to the dashboard if you're an admin and preference survey if you're a regular user. If you're not logged in clicking the logo brings you to the home page not the login page

Comment (Adamk8) [reply src/components/Nav.tsx]: Did we want to just route admins directly to /dashboard, users to /survey, and not logged in to / ? its kinda odd seeing the login page show up for half a second before it redirects

Comment (nicudy) [reply src/components/Nav.tsx]: Yep, sounds good!

**Review by cbuchko — APPROVED** [review]
nice!

**Review by Adamk8 — APPROVED** [review]

**Review by jayden-chan — APPROVED** [review]

**Pull request merged**